packages/core-components/src/components/Status/Status.tsx

```diff
@@ -32,123 +38,185 @@ const useStyles = makeStyles(
   theme => ({
     status: {
       fontWeight: theme.typography.fontWeightMedium,
-      '&::before': {
-        width: '0.7em',
-        height: '0.7em',
-        display: 'inline-block',
-        marginRight: theme.spacing(1),
-        borderRadius: '50%',
-        content: '""',
-      },
+      alignItems: 'baseline',
```

Comment:
@debsmita1 

<img width="191" alt="image" src="https://github.com/user-attachments/assets/b5b0a032-62cc-4df1-9690-644961aa043c">

Why the texts are a bit below with the icon?